### DISCUSSION TITLE
Suspended account and no feedback

### DISCUSSION THREAD

**Discussion by BKyovtorov21** [community/community#119746] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,
My GitHub account (https://github.com/BKKyovtorov21) was suspended two months ago due to outlook account issues. I can confirm now that the email is secure and would like to proceed with unsuspending my account. The reason for writing here is because of the negative feedback I have received about any of the tickets.

**Comment by ebndev** [comment]
Hi @BKyovtorov21,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
